src/binding/defaultBindings/style.js

```diff
@@ -12,6 +12,12 @@ ko.bindingHandlers['style'] = {
             if (jQueryInstance) {
                 jQueryInstance(element)['css'](styleName, styleValue);
             } else {
+                // Is styleName a custom CSS property?
```

Comment:
Can you reformat this as `else if (..) {` and then add a new `else` block below?